src/com/facebook/buck/jvm/groovy/GroovyLibraryDescription.java

```diff
@@ -99,7 +106,14 @@ public Arg createUnpopulatedConstructorArg() {
                 /* additionalClasspathEntries */ ImmutableSet.<Path>of(),
                 new GroovycToJarStepFactory(
                     groovyBuckConfig.getGroovyCompiler().get(),
-                    args.extraArguments),
+                    args.extraGroovycArguments,
+                    JavacOptionsFactory.create(
+                        defaultJavacOptions,
+                        params,
+                        resolver,
+                        pathResolver,
+                        args
+                    )),
```

Comment:
both of them in this case, I guess (from context that looks like the rule; will see if I can persuade intellij to do this for me in future)